Small correction

WhatWhatz opened the pull request:
when h(n)=0 for the heuristic search the search behaves like the Dijkstra-Algorithmus and not the Breadth-first search.

WhatWhatz committed:
Small correction

when h(n)=0 for the heuristic search the search behaves like the Dijkstra-Algorithmus and not the Breadth-first search.

hofbi reviewed (changes requested):
Could you please give a reference where to prove this statement.

According to the book [Mensch-Maschine-Kommunikation](https://link.springer.com/content/pdf/10.1007%2F978-3-642-05457-0.pdf) at page 78 this seems to be correct:

> Für den Fall, dass h(n)≡0 gewählt wird und g(n) stets gleich der Tiefe des Knotensnist, ist der A-Algorithmus identisch mit der Breitensuche

WhatWhatz commented:
I couldn't find any paper or webpage mentioning the A-algorithm discussed. However, I m pretty sure that for h(n)=0 the A*-search becomes a Dijkstra-search (Ref: http://theory.stanford.edu/~amitp/GameProgramming/Heuristics.html )
I will try asking the professor about it.

WhatWhatz commented:
I asked around and got an answer for h(n)=0 the A and A* algorithms will behave as a Breadth-search if g(n) is defined as the level of the element in the tree. If defined differently the search will behave as a Djikstra-algorithm. (Gonna make corresponding changes)

WhatWhatz committed:
Correction about h(n)=0

If A or A* algorithms have a h(n)=0 the search will behave as a Dijkstra-search. If g(n) is defined at the same time as the level in tree the algorithm will behave as a Breadth-search.
